components/brave_rewards/resources/img/contribute_disabled.svg

```diff
@@ -0,0 +1 @@
+<svg width="104" height="77" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M76.863 50l9.799-6.414c3.146-2.059 6.796 1.694 5.175 5.284-.738-.031-1.37.03-2.073.14l-6.44.495-6.461.495zM29.399 63.022L3.085 73.987c-.02.008-.072.03-.138-.014-.065-.044-.065-.102-.065-.123V43.029c0-.289.17-.546.433-.655l15.137-6.326a.607.607 0 0 1 .478.001c.089.038.252.133.337.344l10.47 25.802a.638.638 0 0 1-.338.827zm68.453-8.997a1.849 1.849 0 0 0-.01-.047c-.494-2.214-1.768-3.805-3.584-4.634 2.108-5.936-4.18-11.502-9.508-8.212-13.891 8.579-13.302 8.11-14.15 9.054l-1.246.09c.31-1.992-.148-4.5-2.836-6.405-10.584-7.5-12.467-9.151-15.673-9.872-9.404-2.135-11.343-2.501-16.651-1.378a72.718 72.718 0 0 0-12.016 3.68l-.534-1.35c-.734-1.853-2.77-2.736-4.55-1.972l-14.89 6.378C.863 39.931 0 41.274 0 42.78v31.08c0 2.24 2.187 3.746 4.161 2.903l25.887-11.059c1.777-.76 2.627-2.884 1.896-4.733l-.512-1.292 1.562-.67c3.021-1.294 6.736-.712 9.937 1.556 9.007 6.383 9.122 6.671 11.428 7.553.744.285 1.569-.112 1.842-.886.274-.774-.107-1.632-.851-1.917-1.735-.664-1.635-.721-10.803-7.218-4.002-2.836-8.73-3.527-12.646-1.849l-1.562.67L23.27 39.06a69.883 69.883 0 0 1 11.495-3.513c3.986-.843 6.1-.937 15.441 1.364 2.723.675 3.896 1.775 14.696 9.428 1.625 1.152 2.181 2.887 1.133 4.86-.12.224-1.23 2.163-3.334 1.083-3.357-1.724-11.362-5.793-11.443-5.834-.713-.362-1.572-.055-1.92.686-.348.741-.052 1.635.66 1.997.08.041 8.08 4.108 11.433 5.829 2.635 1.353 5.15.407 6.655-1.599 29.664-2.146 21.346-1.636 24.042-1.604 1.318.24 2.49 1.018 2.912 2.872.647 3.471-2.773 4.642-3.57 4.866l-29.555 6.177c-.778.162-1.28.95-1.125 1.758.137.71.736 1.2 1.406 1.2.094 0 .189-.01.284-.03.063-.013 29.618-6.189 29.68-6.205 2.574-.7 6.67-3.251 5.692-8.371z" fill="#E3AFD8"/><path d="M57.35 15.602c3.538-3.561 9.465-2.754 11.937 1.62.32.566 1.469 1.607 2.91 1.574 1.153-.027 2.185-.722 2.752-1.636 1.103-1.782 2.442-2.735 3.63-3.24 2.637 4.693 8.26 8.264 10.386 8.965-.58 5.037-3.46 9.771-8.58 14.097a42.112 42.112 0 0 1-8.26 5.411c-8.274-4.024-16.98-11.7-16.99-21.41a7.578 7.578 0 0 1 2.215-5.381zm15.994 30.284c1.512-.72 5.454-2.758 9.395-6.087 6.386-5.395 9.739-11.47 9.98-18.07 3.32-1.943 5.643-4.894 7.844-7.3 2.503-2.735 3.566-6.988 2.503-5.924-1.064 1.063-3.19-3.19-4.253 2.126-1.438 2.877-6.15 7.034-9.094 8.555-4.574-2.311-9.244-6.578-9.249-11.763-.004-3.833 5.066-5.132 6.927-1.84.916 1.618 3.478 1.85 4.626-.01 1.73-2.795 5.936-2.48 6.521.145 1.332 5.976.269 5.272.269 3.85 0-3.19 3.611-.255 4.253-1.063.656-.827-.088-2.746-1.266-4.467-2.959-4.325-8.652-5.481-12.086-1.5C84.947-2.96 74.756.89 77.203 10.51c-1.678.699-3.546 1.974-5.096 4.295-3.859-5.958-12.261-6.922-17.345-1.803a11.244 11.244 0 0 0-3.286 7.985c.011 11.564 10.05 20.446 19.442 24.903a2.824 2.824 0 0 0 2.426-.004z" fill="#FF8D71"/></g></svg>
```

Comment:
where do we use this?

Reply:
Seems to be not used anymore. Will be deleted.